internals/daemon/daemon.go

```diff
@@ -109,10 +109,10 @@ type Daemon struct {
 }
 
 // XXX Placeholder for now.
```

Comment:
If we are exporting this, this shouldn't be a placeholder. I'd be reluctant to just strip it since it's used everywhere and seems valuable to store e.g. authenticated user information in the future. Curious to hear Ben's and Gustavo's opinion on this one, but I'd just include a brief comment actually explaining why this exists.

See this snippet from the client code:
https://github.com/canonical/pebble/blob/00bcd1f7dda570bcdf42600545ddc1707d24365e/client/client.go#L384-L387

Reply:
You're talking about UserState right?  And yes at least in snapd it is used to persist authentication information across requests, so I will mention this.